ICAFIX/scripts/MapVolumeToCIFTI.sh

```diff
@@ -0,0 +1,179 @@
+StudyFolder="${1}"
```

Comment:
QuNex might like us to source debug.shlib (and test that it still works afterwards), even if we don't use newopts in an internal script.  If it also passes `set -eu` cleanly, that would be nice.

Reply:
Does this suggest I should source debug.shlib and add `set -eu` to this script?

Reply:
I have added a suggested change.